iree_tests/conftest.py

```diff
@@ -13,6 +13,7 @@
 import os
 import pytest
 import subprocess
+from ireers import *
```

Comment:
How about putting `ireers` in a `common/` folder?

Current (reaching deep into a sibling):
```
iree_special_models/
  ireers/
  sd3/
    (depends on ../ireers/)
iree_tests
  (depends on ../iree_special_models/ireers)
```

Other options:

*
    ```bash
    # This puts "common" at the root, possibly shared with e2eshark
    common/
      fixtures.py
    iree_special_models/
    iree_tests/
    ```

*
    ```bash
    # This keeps the conftest.py at the root, which might not be wanted
    iree_tests/
      common/
      onnx/
      pytorch/
      special_models/
    ```

*
    ```bash
    # This excludes special_tests from conftest.py. Need a name for non-special tests
    iree_tests/
      common/
      tests/
        onnx/
        pytorch/
      special_tests/
        sd3/
    ```

Naming is hard :P

Edit: Could keep `ireers/` instead of `common/` (or nested under `common/`, to keep it installable.